packages/salesforcedx-apex-debugger/src/adapter/apexDebug.ts

```diff
@@ -4,10 +4,11 @@
  * Licensed under the BSD 3-Clause license.
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
+/* eslint-disable @typescript-eslint/no-misused-promises */
 
+import { Connection, Org } from '@salesforce/core';
 import { ConfigAggregator } from '@salesforce/core/configAggregator';
```

Comment:
`ConfigAggregator` can also be imported from the root `@salesforce/core`.

```suggestion
import { Connection, Org, ConfigAggregator } from '@salesforce/core';
```